libfreerdp/core/update.c

```diff
@@ -1653,16 +1854,20 @@ BOOL update_read_suppress_output(rdpUpdate* update, wStream* s)
 	return TRUE;
 }
 
-static void update_send_set_keyboard_indicators(rdpContext* context, UINT16 led_flags)
+static BOOL update_send_set_keyboard_indicators(rdpContext* context, UINT16 led_flags)
 {
 	wStream* s;
 	rdpRdp* rdp = context->rdp;
+	BOOL ret;
 
 	s = rdp_data_pdu_init(rdp);
+	if (!s)
```

Comment:
rdp_data_pdu_init initializes the stream with 2048 byte. We directly only write 4 plus headers. So I thought it's not really required here.